### PR TITLE
Feature - Property API Property ID Fix

### PR DESCRIPTION
**Pull request by a-jomills**
Summary:
This PR adds text specifying the restriction set on Provider Property Id.

### CODE REVIEW

**Comment by a-jomills** [comment]
[X:  Validate ProviderPropertyId is ASCII AlphaNumeric](https://trello.com/c/cB2ZT3We/831-x-validate-providerpropertyid-is-ascii-alphanumeric)

File: files/swagger_propertyApi.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
 				"parameters": [{
 						"name": "providerPropertyId",
 						"in": "path",
-						"description": "The provider's unique ID for property.",
+						"description": "The provider's unique ID for property. Allowable characters include ASCII a-z,A-Z, 0-9, Underscore, Dash.",
```
Comment (jgorman-exp) [inline files/swagger_propertyApi.json]: There is no need to duplicate these details here on the GET request.  This information is only needed on the Property model definition.  Please revert this line.

File: files/swagger_propertyApi.json [hunk 2]
```diff
@@ -71,7 +71,7 @@
 					}, {
 						"name": "providerPropertyId",
 						"in": "path",
-						"description": "The provider's unique ID for property.",
+						"description": "The provider's unique ID for property. Allowable characters include ASCII a-z,A-Z, 0-9, Underscore, Dash.",
```
Comment (jgorman-exp) [inline files/swagger_propertyApi.json]: Please revert.

File: files/swagger_propertyApi.json [hunk 3]
```diff
@@ -160,7 +160,7 @@
 					}, {
 						"name": "providerPropertyId",
 						"in": "path",
-						"description": "The provider's unique property ID for property.",
+						"description": "The provider's unique property ID for property. Allowable characters include ASCII a-z,A-Z, 0-9, Underscore, Dash.",
```
Comment (jgorman-exp) [inline files/swagger_propertyApi.json]: Please revert.

File: files/swagger_propertyApi.json [hunk 4]
```diff
@@ -490,7 +490,8 @@
 			"type": "object",
 			"properties": {
 				"providerPropertyId": {
-					"type": "string"
+					"type": "string",
+					"description": "The provider's unique ID for property. Allowable characters include ASCII a-z,A-Z, 0-9, Underscore, Dash."
```
Comment (jgorman-exp) [inline files/swagger_propertyApi.json]: Please add a space after each comma, and lowercase "Underscore" and "Dash."  In the .md file, too.

File: files/swagger_propertyApi.json [hunk 5]
```diff
@@ -656,7 +657,8 @@
 					"type": "string"
 				},
 				"providerPropertyId": {
-					"type": "string"
+					"type": "string",
+					"description": "The provider's unique ID for property. Allowable characters include ASCII a-z,A-Z, 0-9, Underscore, Dash."
```
Comment (jgorman-exp) [inline files/swagger_propertyApi.json]: Please revert.

Comment (a-jomills) [inline files/swagger_propertyApi.json]: :+1:

File: files/swagger_propertyApi.json (hunk 4 above)
Comment (a-jomills) [inline files/swagger_propertyApi.json]: :+1:

File: files/swagger_propertyApi.json (hunk 3 above)
Comment (a-jomills) [inline files/swagger_propertyApi.json]: :+1:

File: files/swagger_propertyApi.json (hunk 2 above)
Comment (a-jomills) [inline files/swagger_propertyApi.json]: :+1:

File: files/swagger_propertyApi.json (hunk 1 above)
Comment (a-jomills) [inline files/swagger_propertyApi.json]: :+1:

**Pull request merged**